dev_guide/app_tutorials/quickstarts.adoc

```diff
@@ -0,0 +1,27 @@
+= Quickstart Templates
+
+A quickstart is a basic example for getting an application running on OpenShift using various languages and frameworks.  Quickstarts are defined in a link:../templates.html[template] which defines a set of components that make up the application (eg Services, Build Configurations, Deployment Configurations, etc).  The template references images and source repositories needed to build and deploy the application.
+
+The most basic way to explore a quickstart is to simply instantiate its template.  Your administrator may have already installed these templates in your OpenShift cluster, in which case you can simply select it from the Web console.  See the link:../templates.html[template] documentation for more details on how to instantiate a template.
+
+Many quickstarts include a reference to a source repository that contains the application source code.  To customize the quickstart, you will need to fork the repository and then, when instantiating the template, substitute the default source repository name with your forked repository.  This will result in builds that are performed using your source code instead of the provided example source.  You can then update the code in your source repository and launch a new build to see the changes reflected in the deployed application.
+
+== Web Framework Quickstart Templates
+
+These quickstarts provide a minimum application implemented in the framework and language indicated.
+
+* CakePHP - a PHP web framework (includes a MySQL database)
+** link:https://github.com/openshift/origin/tree/master/examples/Quickstarts/cakephp-mysql.json[Template definition]
```

Comment:
Looks like all the links to the json files have `Quickstarts` in the path but should be lowercase `quickstarts`.